Mathlib/Computability/TuringMachine.lean

```diff
@@ -674,7 +674,7 @@ attribute [local simp] Respects TM2.step TM2.stepAux trNormal
 theorem tr_respects : Respects (TM2.step M) (TM1.step (tr M)) TrCfg := by
   -- Porting note (https://github.com/leanprover-community/mathlib4/issues/12129): additional beta reduction needed
   intro c₁ c₂ h
-  cases' h with l v S L hT
+  obtain @⟨l, v, S, L, hT⟩ := h
```

Comment:
Can you combine with the the `rcases` on the next line, please?

Reply:
Done.